### DISCUSSION TITLE
RFE: Immediately show artifacts on summary when they are uploaded

### DISCUSSION THREAD

**Discussion by pbrezina** [community/community#13260] [Actions] [answered]
I have a matrix job with `fail-fast` off, each job runs for a long time (~hour). Currently, the artifacts are visible on the "Summary" tab only when all jobs are finished.

I would like it to be present immediately when an artifact is available, so I can start debugging failed job sooner instead of waiting for the whole workflow to complete.

**Comment by ethomson** [comment, marked as answer]
Thanks @pbrezina - this is a good suggestion. We'll put this on the backlog.
